### ISSUE TITLE
nvalid value at 'track_config.releases[0].status' 

### ISSUE THREAD

**Issue by vytautas-pranskunas-** [r0adkll/upload-google-play#81]
Hello,

recently I have started to get this error: 
```
Run r0adkll/upload-google-play@v1
Warning: WARNING!! 'releaseFile' is deprecated and will be removed in a future release. Please migrate to 'releaseFiles'.
Creating a new Edit for this release
Validating track 'alpha'
Uploading app-release.aab
Successfully uploaded 1 artifacts
Adding 1 artifacts to release on 'alpha' track
Error: Invalid value at 'track_config.releases[0].status' (type.googleapis.com/google.play.publishingapi.v3.Release.Status), ""
```

my script looke like:
```
- name: Release app to alpha track
      uses: r0adkll/upload-google-play@v1
      with:
        serviceAccountJsonPlainText: ${{ secrets.PLAYSTORE_ACCOUNT_KEY }}
        packageName: com.senio.assist
        releaseFile: app-release.aab
        track: alpha
        whatsNewDirectory: distribution/whatsnew
```

I did not get it earlier. 
Any ideas why it so?

Thanks

**Comment by TinasheMzondiwa** [comment]
Experiencing the same error.

**Comment by rynld** [comment]
I have a similar issue:

`Invalid value at 'track_config.releases[0].status' (type.googleapis.com/google.play.publishingapi.v3.Release.Status), ""`

My config is:
```
 - uses: r0adkll/upload-google-play@v1
        with:
          serviceAccountJson: service_account.json
          packageName: com.myapp.name
          releaseFiles: build/app/outputs/bundle/release/app-release.aab
          track: internal
```

The issue started happening on September 21th.

**Comment by yairm210** [comment]
Encountering the same problem 
Seems to be a problem in addReleasesToTrack, using androidPublisher.edits.tracks.update()

Going into the googleApis and reading through their gaziliion releases, it appears that they [updated the androidPublisher](https://github.com/googleapis/google-api-nodejs-client/releases/tag/androidpublisher-v2.0.0) together with many other of their libs, and this is a major version (2.0.0) hence the "Breaking changes - update the API" comment there

However...
```
~$ npm info @googleapis/androidpublisher

@googleapis/androidpublisher@0.2.0 | Apache-2.0 | deps: 1 | versions: 1
androidpublisher
https://github.com/googleapis/google-api-nodejs-client

keywords: google

dist
.tarball: https://registry.npmjs.org/@googleapis/androidpublisher/-/androidpublisher-0.2.0.tgz
.shasum: b6fed0fb25511b2ee4f2e1bfb8531a3ca9d70f37
.integrity: sha512-aOSzJBqgYY1WIbJpZRfTUzJ1wZyq1luIVHRSzM+bPVUPcfGdz8ZQTmU2PuNiA1B/3643i05hhlX4n8a2NCUH7g==
.unpackedSize: 894.0 kB

dependencies:
googleapis-common: ^5.0.1 

maintainers:
- bcoe <bencoe@gmail.com>
- google-wombot <node-team-npm+wombot@google.com>
- justinbeckwith <justin.beckwith@gmail.com>

dist-tags:
latest: 0.2.0  

published 6 months ago by google-wombot <node-team-npm+wombot@google.com
```

It appears there [was a bug](https://github.com/googleapis/google-api-nodejs-client/issues/2750) preventing the submodules from being published.
So now what?
Well, the bug was fixed a week ago, so the new submodule versions should be rolling out soon (I hope)
In the meantime, since the code of the release [is available from the official release](https://github.com/googleapis/google-api-nodejs-client/archive/refs/tags/androidpublisher-v2.0.0.zip), we could take that and use it, but I'm not sure about potential errors.

**Comment by yairm210** [comment]
No, wait, my mistake
I see you're using googleApis directly, so there should be no problem:

```
~$ npm info googleapis

googleapis@87.0.0 | Apache-2.0 | deps: 2 | versions: 198
Google APIs Client Library for Node.js
https://github.com/googleapis/google-api-nodejs-client#readme

keywords: google, api, google apis, client, client library

dist
.tarball: https://registry.npmjs.org/googleapis/-/googleapis-87.0.0.tgz
.shasum: b4fabbc4d7496491a67ac531b698d16983c00c8f
.integrity: sha512-/SfRpfshQoluC309qRkNWUiTkUN5uQtP/TuPvYbw/7khiJJnxrhP632zPZPKT+8zEPe5/j5hYzMnJKywJNeFvw==
.unpackedSize: 92.5 MB

dependencies:
google-auth-library: ^7.0.2 googleapis-common: ^5.0.2   

maintainers:
- justinbeckwith <justin.beckwith@gmail.com>
- google-wombot <node-team-npm+wombot@google.com>

dist-tags:
latest: 87.0.0        legacy-8: 48.0.0      next: 37.0.0-webpack  

published a week ago by google-wombot <node-team-npm+wombot@google.com>
```

**Comment by vytautas-pranskunas-** [comment]
@yairm210 who is using google apis directly? did not get anything...
If there is a bug do you know when it is fixed?

**Comment by yairm210** [comment]
Or it could be the other issue in this repo for the same problem that provides a solution as well...

tl;dr, This was a version bug in 1.0.16, temp solution is to change to `@v1.0.15` instead of `@v1`

[Tested, works](https://github.com/yairm210/Unciv/runs/3712736558).

@r0adkll close this issue please, it's a dupe of #80
